assets/xml/objects/object_bdan_objects.xml

```diff
@@ -0,0 +1,61 @@
+        <Texture Name="gJabuObjectsTex_07348" OutName="jabu_objects_tex_00007348" Format="ci8" Width="32" Height="64" Offset="0x7348"/>
+        <Texture Name="gJabuObjectsTex_08D10" OutName="jabu_objects_tex_00008D10" Format="rgb5a1" Width="32" Height="64" Offset="0x8D10"/>
+        <Texture Name="gJabuObjectsTex_09D10" OutName="jabu_objects_tex_00009D10" Format="rgb5a1" Width="32" Height="32" Offset="0x9D10"/>
+        <Texture Name="gJabuObjectsTex_0A510" OutName="jabu_objects_tex_0000A510" Format="rgb5a1" Width="32" Height="64" Offset="0xA510"/>
+        <Texture Name="gJabuObjectsTex_0B510" OutName="jabu_objects_tex_0000B510" Format="rgb5a1" Width="32" Height="32" Offset="0xB510"/>
+        <Texture Name="gJabuObjectsTex_0BD10" OutName="jabu_objects_tex_0000BD10" Format="rgb5a1" Width="32" Height="32" Offset="0xBD10"/>
+        <Texture Name="gJabuObjectsTex_0C510" OutName="jabu_objects_tex_0000C510" Format="rgb5a1" Width="32" Height="32" Offset="0xC510"/>
+
+        <!-- Palettes -->
+        <Texture Name="gJabuObjects1TLUT" OutName="jabu_objects_1_tlut" Format="rgb5a1" Width="160" Height="1" Offset="0x0000"/>
+        <Texture Name="gJabuObjects2TLUT" OutName="jabu_objects_2_tlut" Format="rgb5a1" Width="160" Height="1" Offset="0x0660"/>
+        <Texture Name="gJabuObjects3TLUT" OutName="jabu_objects_3_tlut" Format="rgb5a1" Width="160" Height="1" Offset="0x0CC0"/>
+        <Texture Name="gJabuObjects4TLUT" OutName="jabu_objects_4_tlut" Format="rgb5a1" Width="160" Height="1" Offset="0x1320"/>
+        <Texture Name="gJabuObjects5TLUT" OutName="jabu_objects_5_tlut" Format="rgb5a1" Width="160" Height="1" Offset="0x1980"/>
+        <Texture Name="gJabuObjects6TLUT" OutName="jabu_objects_6_tlut" Format="rgb5a1" Width="160" Height="1" Offset="0x1FE0"/>
+        <Texture Name="gJabuObjects7TLUT" OutName="jabu_objects_7_tlut" Format="rgb5a1" Width="160" Height="1" Offset="0x2640"/>
+        <Texture Name="gJabuObjects8TLUT" OutName="jabu_objects_8_tlut" Format="rgb5a1" Width="160" Height="1" Offset="0x2CA0"/>
+        <Texture Name="gJabuObjects9TLUT" OutName="jabu_objects_9_tlut" Format="rgb5a1" Width="204" Height="1" Offset="0x3300"/>
+        <Texture Name="gJabuObjects10TLUT" OutName="jabu_objects_10_tlut" Format="rgb5a1" Width="16" Height="16" Offset="0x39B0"/>
+        <Texture Name="gJabuObjects11TLUT" OutName="jabu_objects_11_tlut" Format="rgb5a1" Width="16" Height="16" Offset="0x6540"/>
+
+        <!-- DLists -->
+        <DList Name="gJabuObjectsDL_00590" Offset="0x0590"/>
+        <DList Name="gJabuObjectsDL_00BF0" Offset="0x0BF0"/>
+        <DList Name="gJabuObjectsDL_01250" Offset="0x1250"/>
+        <DList Name="gJabuObjectsDL_018B0" Offset="0x18B0"/>
+        <DList Name="gJabuObjectsDL_01F10" Offset="0x1F10"/>
+        <DList Name="gJabuObjectsDL_02570" Offset="0x2570"/>
+        <DList Name="gJabuObjectsDL_02BD0" Offset="0x2BD0"/>
+        <DList Name="gJabuObjectsDL_03230" Offset="0x3230"/>
```

Comment:
These all seem to be Jabu door sections.

Reply:
To follow up on this, `z_door_shutter.c` contains this array:
```c
static Gfx* D_809982B4[] = {
    0x06000590, 0x06000BF0, 0x06002BD0, 0x060018B0, 0x06001F10, 0x060018B0, 0x06001250, 0x06000BF0,
};
```
which seem to be these DLists; they appear elsewhere in that file too.